roles/validate_operator_bundle/tasks/main.yml

```diff
@@ -59,7 +59,7 @@
 
     - name: "Fail if Pyxis response is not 200"
       fail:
-        msg: "Error collecting OCP version range from Pyxis.  Error message: {{ supported_indices_json.content }}"
+        msg: "Error collecting OCP version range from Pyxis - status {{ supported_indices_json.status_code }}.  Error message: {{ supported_indices_json.content }}"
```

Comment:
I cannot see code coverage the line. You can easily get code different than 200 
with typo in ocp version.

BTW you need't cover all test-cases in midstream ci.
It be simpler to test in upstream ci.
https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/blob/master/.github/workflows/build_and_test_upstream_ci_image.yml

Because you can "mock" different values of the "com.redhat.openshift.versions" label with modification of the {{ work_dir }}/bundle-skopeo-inspect.json file

Adding more images for different value of the `com.redhat.openshift.versions` does not scale

Reply:
I'm a bit confused here.  Isn't that workflow (as well as the `build_and_test_midstream_ci_image.yml` workflow) used to test the CI image itself and not the playbook code?

Reply:
We do not have any CI image. There is just midstream image which is published.
It is just a way run "tests" in upstream on fedora in github actions